StardewCropCalc/src/com/elizabethburgessamt/stardewcropcalc/seeds/Seed.java

```diff
@@ -0,0 +1,48 @@
+package com.elizabethburgessamt.stardewcropcalc.seeds;
+
+public class Seed {
+
+	private String crop;
+	private int growth;
+	private int regrowth;
+	private int cost;
+	private int sell;
+	private boolean trellis;
+	
+	public Seed(String crop, int growth, int regrowth, int cost, int sell, boolean trellis){
+		this.crop = crop;
+		this.growth = growth;
+		this.regrowth = regrowth;
+		this.cost = cost;
+		this.sell = sell;
+		this.trellis = false;
+		if ("TRUE".equals(trellis)) {
+			this.trellis = true;
+		}

```

Comment:
This will fail because a String won't .equal a boolean.  I gave the code to you to show you how you'd want to compare Strings, not to say this is actually what you should do.  So to be clear on this:
this.trellis = trellis;

Reply:
You shouldn't put code in that you don't understand.  You'll probably respond that you thought you did, but you still need to use your head on this.  Strings are Objects, booleans are not, they won't resolve the same way.  You haven't used Java in a long time so it's doubly important that you make sure you understand everything.